libs/testing/src/user.session.ts

```diff
@@ -241,13 +239,16 @@ export class UserSession {
     );
 
     this.token = `Bearer ${response.body.data}`;
-    this.testAgent = superAgentDefaults(request(this.requestEndpoint)).set('Authorization', this.token);
+    this.testAgent = superAgentDefaults(request(this.requestEndpoint))
+      .set('Authorization', this.token)
+      .set('Novu-Environment-Id', this.environment._id);
   }
 
   private async fetchJwtEE() {
     await this.updateEETokenClaims({
       externalId: this.user ? this.user._id : '',
       externalOrgId: this.organization ? this.organization._id : '',
+      // TODO: @adamchmara, is this necessary anymore?
       environmentId: this.environment ? this.environment._id : '',
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** the `environmentId` line shouldn't be needed anymore if its also removed from community.